pkgs/development/tools/misc/grcov/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, rustPlatform, fetchFromGitHub, llvmPackages }:
```

Drop unused argument

```suggestion
{ lib, rustPlatform, fetchFromGitHub }:
```